crates/rooch-rpc-client/src/lib.rs

```diff
@@ -49,7 +52,7 @@ impl ClientBuilder {
         self
     }
 
-    pub async fn build(self, http: impl AsRef<str>) -> Result<Client> {
+    async fn build(self, http: impl AsRef<str>, ethereum_url: &str) -> Result<Client> {
```

Comment:
rooch RPC server provide both: rooch API and ethereum API, so we do not need another url.

Reply:
Ok. I'll unify them